<ui version="4.0" >
 <class>CodecListSelecterFormClass</class>
 <widget class="QDialog" name="CodecListSelecterFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CodecListSelecterForm</string>
  </property>
  <widget class="QComboBox" name="comboBoxLang1" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>20</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 1</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang2" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>50</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 2</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang3" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 3</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 4</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang4" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>110</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 5</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang5" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>140</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang6" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang7" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 9</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang9" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>260</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 8</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 6</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang8" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLang10" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>290</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 7</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Language 10</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>330</y>
     <width>77</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
